### PR TITLE
Drag drop image

### PR DESCRIPTION
**Pull request by suraj-webkul**
Issue #7157 fixed.

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
This is working fine.

**Review by Sanjaybhattwebkul — DISMISSED** [review]

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/variations.blade.php [hunk 1]
```diff
@@ -410,9 +415,19 @@ class="control"
...
                     let self = this;
 
-                    this.super_attributes.forEach(function (attribute) {
-                        self.variant[attribute.code] = '';
-                    })
+                    setTimeout(() => {
+                        (this.superAttributes).forEach(function (attribute) {
+                            self.variant[attribute.code] = '';
+                        })
+                    }, 1000);
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/variations.blade.php:422]: Improve this.

File: packages/Webkul/Attribute/src/Http/Controllers/AttributeController.php [hunk 2]
```diff
@@ -192,4 +196,22 @@ public function massDestroy()
...
+    public function productSuperAttributes($id) 
+    {
+        $product = $this->productRepository->findOrFail($id);
+
+        $superAttributes = $this->productRepository->getSuperAttributes($product);
+
+        return response()->json([
+            'error' => false,
+            'data'  => $superAttributes
+        ]);
```
Comment (devansh-webkul) [inline packages/Webkul/Attribute/src/Http/Controllers/AttributeController.php:215]: Improve this.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]
Changes.

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**